layouts/profile.tsx

```diff
@@ -8,7 +8,9 @@ const ProfileLayout = ({ children }: { children: React.ReactNode }) => {
     <div className="flex flex-col min-h-screen">
       <TopNav />
       <div className="page-container flex grow flex-col items-center">
-        <main className="flex pb-16 w-full flex-1 flex-col items-center bg-light-slate-2">{children}</main>
+        <main className="container px-2 flex md:px-16 pb-16 w-full flex-1 flex-col items-center bg-light-slate-2">
```

Comment:
### REGRESSION!!!

I see that the changes to the `main` occured in the `ProfileLayout` component, this will lead to Regression in other components where the `ProfileLayout` is integrated. See pages/component where regression occurs below with screenshots

- The `/user/[username]` page integrates the `ProfileLayout` and it as regressed: https://deploy-preview-1490--oss-insights.netlify.app/user/babblebey/highlights

![image](https://github.com/open-sauced/insights/assets/25631971/e8fab133-7fe9-45c8-ae51-5b5e5de097eb)

- The `/user/setting` page is the second component that integrates the `ProfileLayout` and it as also regressed: https://deploy-preview-1490--oss-insights.netlify.app/user/settings

![image](https://github.com/open-sauced/insights/assets/25631971/e9977a11-1426-49de-ad78-20cd73320e41)

This is due to the classes (`container`, `px-2` and `md:px-16`) that we moved up into the component.

### FIX!!

```suggestion
        <main className="flex pb-16 w-full flex-1 flex-col items-center bg-light-slate-2">
```

We will have to remove the classes (`container`, `px-2` and `md:px-16`), from there moving it back to its initial place in the child `div`.